py/tools/pex/BUILD.bazel

```diff
@@ -0,0 +1,71 @@
+load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_file")
+load("@bazel_skylib//rules:common_settings.bzl", "bool_flag")
+load("@rules_python//python:defs.bzl", "py_binary")
+load("//py:defs.bzl", _py_binary = "py_binary")
```

Comment:
Perhaps rename one symbol explicitly to `rules_python_py_binary`? It's hard to follow with just the _ rename.